Investigate random subdue test failures

`TestRunExecIntervalSuite` and `TestRunExecCronSuite` intermittently fail when unexpected messages are received when the check is supposed to be subdued. The last few times I noticed this, the failures occurred around 7pm CST, this could be a coincidence or something time related. When this condition is hit, both suites will fail.

```
--- FAIL: TestRunExecIntervalSuite (2.04s)
    --- FAIL: TestRunExecIntervalSuite/TestStart#01 (1.00s)
        Error Trace:    check_scheduler_test.go:152
        Error:          Not equal: 
                        expected: 0
                        actual: 1
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -18,6 +18,10 @@ change that will cause existing Sensu alpha installations to break if upgraded.
 This change was made before beta release so that further breaking changes could
 be avoided.
 
+### Fixed
+- Fixed a bug in time.InWindow that in some cases would cause subdued checks to
+  be executed.
+
 ## [2.0.0-alpha.17] - 2018-02-13
 ### Added
 - Add .gitattributes file with merge strategy for the Changelog.

--- a/util/time/time.go
+++ b/util/time/time.go
@@ -14,7 +14,7 @@ import (
 func InWindow(current time.Time, begin, end string) (bool, error) {
 	// Get the year, month and day of the provided current time (e.g. 2016, 01 &
 	// 02)
-	year, month, day := current.Date()
+	year, month, day := current.UTC().Date()
 
 	// Remove any whitespaces in the begin and end times, for backward
 	// compatibility with Sensu v1 so "3:00 PM" becomes "3:00PM" and satisfies the

